examples/image_captioning/train.py

```diff
@@ -0,0 +1,131 @@
+import argparse
```

Comment:
Please add execution bit (`chmod +x`) and shebang line for scripts expected to be run directly.
(Also for `predict.py` and `download.py`)

Reply:
Fixed.